custom_components/localtuya/cover.py

```diff
@@ -203,6 +203,12 @@ def status_updated(self):
                 self._current_cover_position = 100 - curr_pos
             else:
                 self._current_cover_position = curr_pos
+            if self._current_cover_position == 0:
```

Comment:
Do you want to guard against odd readings, with <= 0 and >= 100 respectively?

Reply:
added some catches from another comment threads sugggestions